db-tests/src/test/java/gov/nasa/jpl/aerie/database/UserSequenceTests.java

```diff
@@ -0,0 +1,209 @@
+      final var updatedRows = statement.executeQuery(
+          //language=sql
+          """
+          UPDATE sequencing.user_sequence
+          SET parcel_id = '%d'
+          WHERE id = %d
+          RETURNING id;
+          """.formatted(parcelId, sequenceId));
+
+      updatedRows.next();
+      assertEquals(sequenceId, updatedRows.getInt("id"));
+    }
+  }
+
+  void updateUserSequenceIsLocked(int sequenceId, boolean isLocked) throws SQLException {
+    try (final var statement = connection.createStatement()) {
+      final var updatedRows = statement.executeQuery(
+        //language=sql
+        """
+        UPDATE sequencing.user_sequence
+        SET is_locked = '%b'
+        WHERE id = %d
+        RETURNING id;
+        """.formatted(isLocked, sequenceId));
+
+      updatedRows.next();
+      assertEquals(sequenceId, updatedRows.getInt(updatedRows.getInt("id")));
+    }
+  }
+
+  ArrayList<UserSequence> getUserSequences(int sequenceId) throws SQLException {
+    try (final var statement = connection.createStatement()) {
+      final var sequences = new ArrayList<UserSequence>();
+      final var res = statement.executeQuery(
+        //language=sql
+        """
+        SELECT id, name, definition, parcel_id, workspace_id, owner
+        FROM sequencing.user_sequence
+        WHERE id = %d
+        """.formatted(sequenceId));
+
+      while (res.next()) {
+        sequences.add(new UserSequence(
+            res.getInt("id"),
+            res.getString("name"),
+            res.getString("definition"),
+            res.getInt("parcel_id"),
+            res.getInt("workspace_id"),
+            res.getString("owner")));
+      }
+      return sequences;
+    }
+  }
+  // endregion
+
+  //region Records
+  record UserSequence(int id, String name, String definition, int parcel_id, int workspace_id, String owner) {}
+  //endregion
+
+  @Test
+  void createUserSequence() throws SQLException {
+    parcelId = createParcel("Test Parcel");
+    workspaceId = createWorkspace("Test Workspace", sequenceUser.name());
+    sequenceId = createUserSequence("Test Sequence", parcelId, workspaceId, sequenceDefinition, sequenceUser.name());
+
+    final var userSequence = getUserSequences(sequenceId);
+
+    final var expectedUserSequence = new ArrayList<UserSequence>(1);
+    expectedUserSequence.add(new UserSequence(sequenceId, "Test Sequence", sequenceDefinition, parcelId, workspaceId, sequenceUser.name()));
+
+    assertEquals(expectedUserSequence, userSequence);
+  }
+
+  @Test
+  void updateUserSequenceParcel() throws SQLException {
+    final var testParcelId = createParcel("Test Parcel 2 ");
+    assertDoesNotThrow(()->assignParcelToSequence(testParcelId, sequenceId));
+  }
+
+  @Test
+  void lockUserSequence() throws SQLException {
+    final var testParcelId = createParcel("Test Parcel 2 ");
+    updateUserSequenceIsLocked(sequenceId, true);
+
+    assertThrows(SQLException.class, ()->assignParcelToSequence(testParcelId, sequenceId));
```

Comment:
Minor, I'd prefer if you catch this and check that the error message is the one we're expecting. This will help catch if the DB code starts throwing an exception for the wrong reason in the future:
```suggestion
    final var ex = assertThrows(SQLException.class, ()->assignParcelToSequence(testParcelId, sequenceId));
    assertTrue(ex.getMessage().contains("Cannot update locked user sequence."));
```